test/e2e/create_cluster_test.go

```diff
@@ -1062,7 +1062,6 @@ func TestCreateClusterRequestServingIsolation(t *testing.T) {
 
 	e2eutil.NewHypershiftTest(t, ctx, func(t *testing.T, g Gomega, mgtClient crclient.Client, hostedCluster *hyperv1.HostedCluster) {
 		guestClient := e2eutil.WaitForGuestClient(t, testContext, mgtClient, hostedCluster)
-		e2eutil.EnsurePSANotPrivileged(t, ctx, guestClient)
```

Comment:
I don't think that we should be removing this function.  Instead, I think we should punt restricted PSA to 4.20 via a change to the function like was done by https://github.com/openshift/hypershift/pull/4875.  Then in a follow-up PR we can restore restricted by default PSA to align with how 4.19 is currently defined https://github.com/openshift/api/blob/release-4.19/features/features.go#L79-L85.